test/conformance/enqueue/urEnqueueKernelLaunchAndMemcpyInOrder.cpp

```diff
@@ -287,9 +339,18 @@ TEST_F(urEnqueueKernelLaunchIncrementMultiDeviceTest, Success) {
         }
     }
 
-    // synchronize on the last queue only, this has to ensure all the operations
+    // While the device(s) execute, loop over the events and if completed, verify the results
+    if (runBackgroundCheck) {
+        this->runBackgroundCheck(Events);
+    }
+
+    // synchronize on the last queue/event only, this has to ensure all the operations
     // are completed
-    urQueueFinish(queues.back());
+    if (waitOnEvent) {
```

Comment:
since the 'background check' function waits for its threads to finish, this is guaranteed to have been completed already. Was that intentional? That'd be odd for an `event wait` test :)